docs/tutorials/11_GMPHDTutorial.py

```diff
@@ -0,0 +1,577 @@
+# found at https://notebook.community/empet/Plotly-plots/Gaussian-Mixture
+#
+from scipy.stats import multivariate_normal
+from mpl_toolkits.mplot3d import Axes3D
+from matplotlib import cm
+
+# This function generates the z values for the Gaussian mixture. It has been edited
+# from a version found at this website https://notebook.community/empet/Plotly-plots/Gaussian-Mixture
+# The website is from this repository https://github.com/empet/Plotly-plots which
+# has a GPL-3.0 License. 
+def get_mixture_density(x, y, weights, means, sigmas):
+
+    # We use the quantiles as a parameter in the multivariate_normal function. We don't need to pass in any quantiles,
+    # but the last axis must have the components x and y
+    quantiles = np.empty(x.shape + (2,))# if  x.shape is (m,n) then  quantiles.shape is (m,n,2)
+    quantiles[:, :, 0] = x
+    quantiles[:, :, 1] = y
+
+    # Go through each gaussian in the list and add its PDF to the mixture
+    z = np.zeros(x.shape)
+    for gaussian in range(len(weights)):
+        z += weights[gaussian]*multivariate_normal.pdf(x=quantiles, mean=means[gaussian,:], cov=sigmas[gaussian])
+    return z
+
+
+# %%
+# For each timestep, create a new figure with 2 subplots. The plot on the left will
+# show the 3D Gaussian Mixture density. The plot on the right will show a 2D
+# birds-eye-view of the space, including the ground truths, detections, clutter, and tracks.
+
+from matplotlib import animation
+from matplotlib import pyplot as plt
+from matplotlib.lines import Line2D # Will be used when making the legend
+
+# This is the function that updates the figure we will be animating. As parametrs we must
+# pass in the elements that will be changed, as well as the index i
+def animate(i, sf, truths, tracks, measurements, clutter):
+    # Set up the axes
+    axL.clear()
+    axR.set_title('Tracking Space at k='+str(i))
+    axL.set_xlabel("x")
+    axL.set_ylabel("y")
+    axL.set_title('PDF of the Gaussian Mixture')
+    axL.view_init(elev=30, azim=-80)
+    axL.set_zlim(0, 0.3)
+
+    # Initialize the variables
+    num_gaussians = len(all_gaussians[i]) # number of functions in this mixture
+    weights = []  # weights of each Gaussian. This is analogous to the probability of its existence
+    means = []    # means of each Gaussian. This is equal to the x and y of its state vector
+    sigmas = []   # standard deviation of each Gaussian.
+
+    # Fill the lists of weights, means, and standard deviations
+    for state in all_gaussians[i]:
+        weights.append(state.weight)
+        means.append([state.state_vector[0], state.state_vector[2]])
+        sigmas.append([state.covar[0][0], state.covar[1][1]])
+    means = np.array(means)
+    sigmas = np.array(sigmas)
+
+    # Generate the z values over the space and plot on the left axis
+    zarray[:,:,i] = get_mixture_density(x, y, weights, means, sigmas)
+    sf = axL.plot_surface(x, y, zarray[:,:,i], cmap=cm.RdBu, linewidth=0, antialiased=False)
```

Comment:
```suggestion
    zarray[:, :, i] = get_mixture_density(x, y, weights, means, sigmas)
    sf = axL.plot_surface(x, y, zarray[:, :, i], cmap=cm.RdBu, linewidth=0, antialiased=False)
```
Whitespace after commas